.github/workflows/pr_master.yml

```diff
@@ -1,14 +1,30 @@
-name: Pr to master
+name: PR to master
+
 on:
   pull_request:
-    branches: master
+    branches: [master] 
+
 jobs:
   tests:
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Install node
+
+      - name: Setup Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '18'
\ No newline at end of file
+          node-version: '18'
+
+      - name: Install E2E dependencies
+        run: npm instal
+        working-directory: ./e2e
+
+      - name: Install Playwright browsers
+        run: npx playwright install
+        working-directory: ./e2e
+
+      - name: Run regression tests
+        run: npm run regression
+        working-directory: ./e2e
```

Comment:
Could you add the same for e2e workflow file

Reply:
Of course. I added this just because I wanted to check these 2 files first